virt/virt-4-8-release-notes.adoc

```diff
@@ -59,6 +57,9 @@ The SVVP Certification applies to:
 //CNV-12348 Add `--proxy-only` option to `virtctl vnc` for only proxying a VNC connection to allow manual connections
 * The `--proxy-only` option for the `virtctl vnc` command allows you to xref:../virt/virt-using-the-cli-tools.adoc#virt-virtctl-commands_virt-using-the-cli-tools[manually connect to a virtual machine instance] through a Virtual Network Client (VNC) connection by using any VNC viewer.
 
+//CNV-9518 Release Note for calico
+** {VirtProductName} supports the use of certified default Container Network Interfaces (CNI) network providers listed in link:https://access.redhat.com/articles/5436171[Certified OpenShift CNI Plug-ins].
```

Comment:
I feel like it might be helpful to point out that these are 3rd party plug-ins that are certified to work with OCP and make the link part of the sentence per our writing guidelines.

Suggestion: {VirtProductName} now supports third-party Container Network Interface (CNI) plug-ins that are [certified by Red Hat](https://access.redhat.com/articles/5436171) for use with {product-title}.

Reply:
Incorporated the suggested wording.